### PR TITLE
Update cards look in Onelive Page

### LINKED ISSUE #858: Update cards look in Onelive Page
**Describe the bug**
In the [Home](https://sefglobal.org), `Our Project` section there are curvy edged cards. In the [One Live ](https://sefglobal.org/onelive/) page, `Take a look at our past One Lives` section there are a different set of cards(not curve edged cards)

**To Reproduce**
1. Go to 'One Live page' (From navigation bar Projects-> Onelive)
2. Scroll down to 'Take a look at our past One Lives'
3. See the inconsistency

**Expected behavior**
One Live `Take a look at our past One Lives` cards need be changed to curve edged cards. So One Live cards will look like home page cards. Use the exact CSS which is used in the Home page curve cards to keep the consistency.

### PR DESCRIPTION
**Pull request by YaiyaSam**
Update cards look in Onelive page

## Purpose

The purpose of this PR is to fix #858

## Goals

Update cards look in Onelive page to make them look similar to the ones in "Our Projects" section in Home page

### Screenshots

image

  
### Preview Link

https://pr-897-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
Google Chrome
Node server
Windows 10

### CODE REVIEW

File: assets/css/global-custom.css [hunk 1]
```diff
@@ -500,6 +504,10 @@ body{
     padding: .5rem .75rem;
 }
 
+.card-height{
+    height: 400px;
+}
+
```
Comment (Gravewalker666) [inline assets/css/global-custom.css]: Pick a better class name, `card-height` doesn't describe what this class actually does. For an example the above class you have created `rounded-24px` name it self is pretty self explanatory. Change the class name into something like that. :)

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css (hunk 1 above)
Comment (YaiyaSam) [reply assets/css/global-custom.css]: I changed it. Could you have a look at it?

Comment (YaiyaSam) [reply assets/css/global-custom.css]: Resolved with commit 86907d68ec4cc3f4f2d824ca8c5db80cc1136a87. please review

**Review by Gravewalker666 — APPROVED** [review]
Great job @YaiyaSam, Thanks for getting this done.
@janethavi can you check it again and merge it?

**Comment by janethavi** [comment]
> Great job @YaiyaSam, Thanks for getting this done.
> @janethavi can you check it again and merge it?

Yes @Gravewalker666 We can!

**Review by janethavi — APPROVED** [review]
Great work @YaiyaSam! Thank you :100:

**Pull request merged**